examples/plot_simulate_gamma.py

```diff
@@ -22,10 +35,12 @@
 hnn_core_root = op.dirname(hnn_core.__file__)
 
 ###############################################################################
-# Then we read the parameters file
+# Read the parameter file and print the between-cell connectivity parameters.
+# Note that these are different compared with the 'default' parameter set used
```

Comment:
Actually they shouldn't be dramatically different. Maybe only the layer2-layer5 connections are turned off and the rest is same? I'd like to move away from importing different param files to initialize the network. Do you think using `default.param` would work?

Reply:
They are: just compare `param['gbar_L*]`. The paper (Law & Jones, 2013) is pretty clear about it too. I haven't tried using the default parameters, though, could be interesting...

Reply:
indeed ... let's leave it like this then